animeal/src/Flows/Main/Modules/Home/Coordinator/HomeCoordinator.swift

```diff
@@ -62,6 +63,8 @@ extension HomeCoordinator: HomeCoordinatable {
 }
 
 extension HomeCoordinator: FeedingPointCoordinatable {
+    var isOverMap: Bool { true }
```

Comment:
I'm not sure about that approach 
@diantyn @ihartsimafeichyk please add your comments

Reply:
Please consider the approach where `FeedingPointDetailsModuleAssembler` takes `isOverMap` param into `init`, and pass it to `FeedingPointDetailsViewModel`. However this could be done in separate PR.

Reply:
done